core/proto/exploration_pb2.py

```diff
@@ -0,0 +1,5792 @@
+# -*- coding: utf-8 -*-
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
+# source: exploration.proto
+"""Generated protocol buffer code."""
```

Comment:
We ought to never check in generated files. Instead, we should rely on the proto generation pipeline to generate this from proto file at resource compilation time prior to the webserver starting. I think this pipeline is already set up for our Oppia-ML protos.

Reply:
Sounds good, once we bring our proto file to oppia-ml repo, I will update the import and remove the current compiled file and it will be good to go.